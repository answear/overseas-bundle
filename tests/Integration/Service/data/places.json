{
    "data": [
        {
            "ZipCode": "44401",
            "Name": "ŠIBINE",
            "ZipcodeName": "HAJTIĆ",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "44402",
            "Name": "VELIKI OBLJAJ",
            "ZipcodeName": "MALI OBLJAJ",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "42253",
            "Name": "PURGA BEDNJANSKA",
            "ZipcodeName": "BEDNJA",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "47273",
            "Name": "SREDNJE PRILIŠĆE",
            "ZipcodeName": "SREDNJE PRILIŠĆE",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "21224",
            "Name": "SLATINE",
            "ZipcodeName": "SLATINE",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "34322",
            "Name": "NURKOVAC",
            "ZipcodeName": "BRESTOVAC",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "20230",
            "Name": "BROCE (PELJEŠAC)",
            "ZipcodeName": "STON (PELJEŠAC)",
            "StandardAvailable": true,
            "CargoAvailable": false
        },
        {
            "ZipCode": "23263",
            "Name": "BANJ (PAŠMAN)",
            "ZipcodeName": "ŽDRELAC (PAŠMAN)",
            "StandardAvailable": true,
            "CargoAvailable": false
        },
        {
            "ZipCode": "52402",
            "Name": "BORUT",
            "ZipcodeName": "CEROVLJE",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "21400",
            "Name": "SUPETAR (BRAČ)",
            "ZipcodeName": "SUPETAR (BRAČ)",
            "StandardAvailable": true,
            "CargoAvailable": false
        },
        {
            "ZipCode": "21400",
            "Name": "MIRCA (BRAČ)",
            "ZipcodeName": "SUPETAR (BRAČ)",
            "StandardAvailable": true,
            "CargoAvailable": false
        },
        {
            "ZipCode": "21403",
            "Name": "SUTIVAN (BRAČ)",
            "ZipcodeName": "SUTIVAN (BRAČ)",
            "StandardAvailable": true,
            "CargoAvailable": false
        },
        {
            "ZipCode": "51302",
            "Name": "POŽAR",
            "ZipcodeName": "KUŽELJ",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "51302",
            "Name": "SEDALCE",
            "ZipcodeName": "KUŽELJ",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "21256",
            "Name": "CISTA PROVO",
            "ZipcodeName": "CISTA PROVO",
            "StandardAvailable": true,
            "CargoAvailable": true
        },
        {
            "ZipCode": "42243",
            "Name": "KORENJAK",
            "ZipcodeName": "MARUŠEVEC",
            "StandardAvailable": true,
            "CargoAvailable": true
        }
    ],
    "status": 0,
    "validations": []
}
