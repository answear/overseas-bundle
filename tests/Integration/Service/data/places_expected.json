[
    {
        "zipCode": "44401",
        "name": "ŠIBINE",
        "zipCodeName": "HAJTIĆ",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "44402",
        "name": "VELIKI OBLJAJ",
        "zipCodeName": "MALI OBLJAJ",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "42253",
        "name": "PURGA BEDNJANSKA",
        "zipCodeName": "BEDNJA",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "47273",
        "name": "SREDNJE PRILIŠĆE",
        "zipCodeName": "SREDNJE PRILIŠĆE",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "21224",
        "name": "SLATINE",
        "zipCodeName": "SLATINE",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "34322",
        "name": "NURKOVAC",
        "zipCodeName": "BRESTOVAC",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "20230",
        "name": "BROCE (PELJEŠAC)",
        "zipCodeName": "STON (PELJEŠAC)",
        "standardAvailable": true,
        "cargoAvailable": false
    },
    {
        "zipCode": "23263",
        "name": "BANJ (PAŠMAN)",
        "zipCodeName": "ŽDRELAC (PAŠMAN)",
        "standardAvailable": true,
        "cargoAvailable": false
    },
    {
        "zipCode": "52402",
        "name": "BORUT",
        "zipCodeName": "CEROVLJE",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "21400",
        "name": "SUPETAR (BRAČ)",
        "zipCodeName": "SUPETAR (BRAČ)",
        "standardAvailable": true,
        "cargoAvailable": false
    },
    {
        "zipCode": "21400",
        "name": "MIRCA (BRAČ)",
        "zipCodeName": "SUPETAR (BRAČ)",
        "standardAvailable": true,
        "cargoAvailable": false
    },
    {
        "zipCode": "21403",
        "name": "SUTIVAN (BRAČ)",
        "zipCodeName": "SUTIVAN (BRAČ)",
        "standardAvailable": true,
        "cargoAvailable": false
    },
    {
        "zipCode": "51302",
        "name": "POŽAR",
        "zipCodeName": "KUŽELJ",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "51302",
        "name": "SEDALCE",
        "zipCodeName": "KUŽELJ",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "21256",
        "name": "CISTA PROVO",
        "zipCodeName": "CISTA PROVO",
        "standardAvailable": true,
        "cargoAvailable": true
    },
    {
        "zipCode": "42243",
        "name": "KORENJAK",
        "zipCodeName": "MARUŠEVEC",
        "standardAvailable": true,
        "cargoAvailable": true
    }
]
